Source/Shared/BPConfiguration.h

```diff
@@ -51,6 +54,7 @@
 @property (nonatomic, strong) NSNumber *numSims;
 @property (nonatomic) BOOL listTestsOnly;
 @property (nonatomic) BOOL quiet;
+@property (nonatomic) int program; // one of BLUEPILL or BP
```

Comment:
Honestly, I don't like the name BP and BLUEPILL because they are confusing. I would like something like BP_MASTER, BP_SLAVE or BP_WORKER, BP_RUNNER.